.env

```diff
@@ -0,0 +1,5 @@
+DB_USER=ilyayudovin

```

Comment:
You do not push this file. You push `.env.example` with dummy info. Add this file to gitignore.